app/controllers/api_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ApiController < ApplicationController
```

Comment:
I love that you abstracted this. However, you will want to add
`self.abstract_class = true`